src/mbgl/map/transform.cpp

```diff
@@ -590,6 +590,23 @@ LatLng Transform::screenCoordinateToLatLng(const ScreenCoordinate& point, LatLng
     return state.screenCoordinateToLatLng(flippedPoint, wrapMode);
 }
 
+std::vector<ScreenCoordinate> Transform::latLngsToScreenCoordinates(const std::vector<LatLng>& latLngs) const {
+    std::vector<ScreenCoordinate> points = state.latLngsToScreenCoordinates(latLngs);
+    for (auto& point : points) {
+        point.y = state.size.height - point.y;
+    }
+    return points;
+}
+
+std::vector<LatLng> Transform::screenCoordinatesToLatLngs(const std::vector<ScreenCoordinate>& points,
+                                                          LatLng::WrapMode wrapMode) const {
+    auto flippedPoints = points;
+    for (auto& flippedPoint : flippedPoints) {
+        flippedPoint.y = state.size.height - flippedPoint.y;
+    }
+    return state.screenCoordinatesToLatLngs(flippedPoints, wrapMode);
```

Comment:
should transform state accept points by value to avoid copy?
`return state.screenCoordinatesToLatLngs(std::move(flippedPoints), wrapMode);`